package.json

```diff
@@ -76,15 +76,16 @@
     "husky": "^8.0.0",
     "jest-image-snapshot": "^5.1.0",
     "lint-staged": "^13.0.3",
-    "prettier": "^2.3.1",
+    "prettier": "^2.8.1",
     "prop-types": "^15.7.2",
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
     "rimraf": "^3.0.2",
     "storybook": "next",
     "ts-jest": "^28.0.8",
-    "typescript": "^4.2.4",
-    "wait-on": "^6.0.0"
+    "typescript": "~4.9.4",
+    "wait-on": "^6.0.0",
+    "webpack": "^5.75.0"
```

Comment:
Is this on purpose?

Reply:
Ah, I thought I got some peer dep warnings, but I think I might be mistaken, will try without.